### PR TITLE
Update as per Feedbacks

### PR DESCRIPTION
**Pull request by sweta1308**
1. Updated the favicon image format for meta data.
image

2. Made the tabs in header clickable.
3. Fetch blogs from hashnode.
image

4. Removed the versioning.
5. Updated icon and icon bg color.
image

6. Updated navbar mobile design.
image

### CODE REVIEW

File: constants/Homepage.constants.ts [hunk 1]
```diff
@@ -13,26 +13,32 @@ export const HEADER_INFO = {
...
     icon: '/images/header-tabs/lineage.svg',
-    name: 'Lineage'
+    name: 'Lineage',
+    link: '/how-to-guides/data-lineage'
   },
   {
     icon: '/images/header-tabs/observability.svg',
-    name: 'Observability'
+    name: 'Observability',
+    link: '/how-to-guides/data-insights'
```
Comment (aniketkatkar97) [inline constants/Homepage.constants.ts:27]: the link is incorrect here, for now lets use `/quality-and-observability`

File: components/BlogsInfo/BlogsInfo.tsx
Comment (aniketkatkar97) [inline components/BlogsInfo/BlogsInfo.tsx:1]: Can't we use same component in docs-v1?

Comment (aniketkatkar97) [reply components/BlogsInfo/BlogsInfo.tsx:1]: why are we adding a new component for this?

File: pages/[version]/index.tsx [hunk 2]
```diff
@@ -28,6 +31,22 @@ export default function Index({ versionsList }: Readonly<Props>) {
   const { isRouteChanging } = useRouteChangingContext();
   const { isMobileDevice } = useNavBarCollapsedContext();
   const { menuItems } = useMenuItemsContext();
+  const [blogPosts, setBlogPosts] = useState([]);
+
+  const fetchPosts = async (): Promise<void> => {
+    try {
```
Comment (aniketkatkar97) [inline pages/[version]/index.tsx:37]: No need of `: Promise<void>` here

File: pages/[version]/index.tsx [hunk 3]
```diff
@@ -74,13 +93,13 @@ export default function Index({ versionsList }: Readonly<Props>) {
...
-                    image={cardInfo.image}
+                {blogPosts.map((cardInfo) => (
+                  <BlogsInfo
+                    image={cardInfo.coverImage.url}
                     key={`${cardInfo.title}${cardInfo.link}`}
-                    link={cardInfo.link}
+                    link={cardInfo.url}
                     title={cardInfo.title}
-                    text={cardInfo.text}
+                    text={cardInfo.brief}
```
Comment (aniketkatkar97) [inline pages/[version]/index.tsx:102]: Let's show skeleton loader while we are fetching the blogs

File: components/BlogsInfo/BlogsInfo.tsx
Comment (sweta1308) [reply components/BlogsInfo/BlogsInfo.tsx:1]: No. As In docs-v1, we are getting the Image component in image property, but in collate, we are getting the url, so, we can use it but we will have to send the component as props.

File: constants/Homepage.constants.ts (hunk 1 above)
Comment (sweta1308) [reply constants/Homepage.constants.ts]: Updated

File: pages/[version]/index.tsx (hunk 2 above)
Comment (sweta1308) [reply pages/[version]/index.tsx]: Updated

File: pages/[version]/index.tsx (hunk 3 above)
Comment (sweta1308) [reply pages/[version]/index.tsx]: Updated

**Review by aniketkatkar97 — DISMISSED** [review]

File: components/TopNav/TopNav.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import { MdMenu, MdMenuOpen } from "react-icons/md";
+import { InstantSearch } from "react-instantsearch";
+import { useDocVersionContext } from "../../docs-v1/context/DocVersionContext";
+import { useNavBarCollapsedContext } from "../../docs-v1/context/NavBarCollapseContext";
+import { SearchContextProvider } from "../../docs-v1/context/SearchContext";
+import { ReactComponent as ApiIcon } from "../../docs-v1/images/icons/api.svg";
+import { ReactComponent as CollateIcon } from "../../images/icons/collate-logo.svg";
+import Search from "../../docs-v1/components/Search/Search";
+import styles from "../../docs-v1/components/TopNav/TopNav.module.css";
+import styles1 from "./TopNav.module.css";
```
Comment (aniketkatkar97) [inline components/TopNav/TopNav.tsx:14]:
```suggestion
import collateStyles from "./TopNav.module.css";
```

File: components/TopNav/TopNav.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+import { useCallback, useEffect, useState } from "react";
+import { MdMenu, MdMenuOpen } from "react-icons/md";
+import { InstantSearch } from "react-instantsearch";
+import { useDocVersionContext } from "../../docs-v1/context/DocVersionContext";
+import { useNavBarCollapsedContext } from "../../docs-v1/context/NavBarCollapseContext";
+import { SearchContextProvider } from "../../docs-v1/context/SearchContext";
+import { ReactComponent as ApiIcon } from "../../docs-v1/images/icons/api.svg";
+import { ReactComponent as CollateIcon } from "../../images/icons/collate-logo.svg";
+import Search from "../../docs-v1/components/Search/Search";
+import styles from "../../docs-v1/components/TopNav/TopNav.module.css";
```
Comment (aniketkatkar97) [inline components/TopNav/TopNav.tsx:13]:
```suggestion
import OMStyles from "../../docs-v1/components/TopNav/TopNav.module.css";
```

File: components/TopNav/TopNav.tsx (hunk 4 above)
Comment (sweta1308) [reply components/TopNav/TopNav.tsx]: Updated

File: components/TopNav/TopNav.tsx (hunk 5 above)
Comment (sweta1308) [reply components/TopNav/TopNav.tsx]: updated

**Review by aniketkatkar97 — DISMISSED** [review]

**Review by aniketkatkar97 — APPROVED** [review]

**Review by aniketkatkar97 — APPROVED** [review]

**Pull request merged**